plugin/output/postgres/postgres.go

```diff
@@ -241,6 +245,17 @@ func (p *Plugin) Start(config pipeline.AnyConfig, params *pipeline.OutputPluginP
 	}
 	p.pool = pool
 
+	stdBackoff := backoff.NewExponentialBackOff()
+	stdBackoff.Multiplier = consts.ExpBackoffDefaultMultiplier
+	stdBackoff.RandomizationFactor = consts.ExpBackoffDefaultRndFactor
+	stdBackoff.InitialInterval = p.config.Retention_
+	stdBackoff.MaxInterval = p.config.Retention_ * 2
+
+	ctxBackoff := backoff.WithContext(stdBackoff, p.ctx)
```

Comment:
If you use 3 variables they should be: 
`stdBackoff` > `expBackoff`
`ctxBackoff` > `expCtxBackoff`
`expBackoff` > `expCtxRetryBackoff`

But I'd recommend use just one  variable `backoff` and modify it.